lib/spack/spack/test/cmd/config.py

```diff
@@ -42,18 +43,35 @@ def test_config_list_scopes():
     output = config("list-scopes")
     assert "command_line" in output
     assert "_builtin" in output
+    assert f"site/{spack.platforms.host()}" in output
```

Comment:
You probably don't want to add this. The session-level fixture `_use_test_platform()` causes the left hand side to be `site/test`.

Reply:
What do you mean by, "session-level?" Is that unconditionally invoked for all tests, or as part of an in-test-file directive, or in test configuration, or on the command line? I'm not currently seeing the invocation mechanism, sorry.

Reply:
They apply to all test functions run during the test "session" (https://docs.pytest.org/en/stable/explanation/fixtures.html).